csharp/EmsApi.Client/V2/Service/EmsApiEndpoints.cs

```diff
@@ -0,0 +1,24 @@
+﻿
+namespace EmsApi.Client.V2
+{
+    /// <summary>
+    /// Well-known API endpoints.
+    /// </summary>
+    public static class EmsApiEndpoints
+    {
+        /// <summary>
+        /// The default endpoint.
+        /// </summary>
+        public const string Default = Efoqa;
+
+        /// <summary>
+        /// The primary production endpoint.
+        /// </summary>
+        public const string Efoqa = "https://ems.efoqa.com/api";
+
+        /// <summary>
+        /// The beta endpoint.
+        /// </summary>
+        public const string Beta = "https://emsapibeta.ausdig.com/api";
+    }
+}
```

Comment:
I went back and forth on this a bit also, but it seems like common practice for some other libs I've used like Amazon (given, the AWS regions are a bit more well-known than our endpoints, but still).

I would like to support the Predix routes as well, but I plan to do it as a separate task because the authentication scheme over there seems to be different as well.